python/tvm/autotvm/task/relay_integration.py

```diff
@@ -49,7 +50,7 @@ def _lower(mod,
                 grc = graph_runtime_codegen.GraphRuntimeCodegen(None, target)
                 grc.codegen(mod["main"])
     # default case
-    compiler = relay.vm.VMCompiler()
+    compiler = runtime.vm.VMCompiler()
```

Comment:
we still want to keep the compiler under the original namespace, but only move the runtime only component

Reply:
So we can just have a relay/backend/vm.py and python/tvm/runtime/vm.py, right?

Reply:
Yes, the idea is that we want the structure to reflect the structure in the C++ project.

More importantly, we should be able to isolate the files under python/tvm/runtime for runtime only packaging.